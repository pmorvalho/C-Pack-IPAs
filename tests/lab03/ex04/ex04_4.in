202 00 303