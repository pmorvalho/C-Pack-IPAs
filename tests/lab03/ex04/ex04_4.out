202 0 303